builtin/providers/aws/resource_aws_api_gateway_api_key.go

```diff
@@ -68,6 +68,15 @@ func resourceAwsApiGatewayApiKey() *schema.Resource {
 				Type:     schema.TypeString,
 				Computed: true,
 			},
+
+			"value": {
+				Type:         schema.TypeString,
+				Optional:     true,
+				Computed:     true,
+				ForceNew:     true,
+				Sensitive:    true,
```

Comment:
@stack72 Are there any other things to do more than this? Not sure how to deal with the state yet